### ISSUE TITLE
Gap in BNB chain data

### ISSUE THREAD

**Issue by m-root** [tradingstrategy-ai/trade-executor#43]
There seems to be missing data in the market price. There is a need to make the market price data gap more flexible inorder to accommodate missing data in the backtests. 

![Image](https://user-images.githubusercontent.com/15989327/198295888-5d6fab99-3b3d-4343-b284-8567857e5494.png)

**Comment by miohtama** [comment]
Let's do two updates (can be separate tickets)

- Increase the default gap to 2 days for all execution where candle time frame <= 1 day (priority 1 - small)
     - This should get rid of this error for now

- Better gap handling (priority 2 - complex)
     - Make the max gap for the strategy execution settable by the user (can be set in the university parameters)
     - and also add a warning about large gaps when the strategy execution is finished

**Comment by miohtama** [comment]
@hieuh25 can guide you where the gap is currently configured (or more like not configured) at the moment.

**Comment by hieuh25** [comment]
> @hieuh25 can guide you where the gap is currently configured (or more like not configured) at the moment.

Yes the function to get price is at: https://github.com/tradingstrategy-ai/trading-strategy/blob/7e332dafb1111ab2818107b2e0f72e8d51a6e500/tradingstrategy/candle.py#L271-L332

Since this function is used in multiple places, I guess it's slightly more delicate to handle than usual. Anyhow we can just leave that function as is and change how we call it depends on situation

**Comment by m-root** [comment]
For the P1 ticket, we can ignore the variables in the package and add a variable with a higher default gap of 2 days at `backtest/backtest_pricing.py` in the function descriptions of `get_sell_price` and `get_buy_price` then pass the values to `return float(self.candle_universe.get_closest_price(pair_id, ts, kind=self.candle_timepoint_kind))` just as a hot fix.

**Comment by m-root** [comment]
Passing all the layers without breaking a thing will be the next headache. It's like a Russian doll.

**Comment by miohtama** [comment]
Mikko is going to work on `BacktestPricingModel` to fix this. CC @AlexTheLion123 @m-root
